lib/expansion_rules.rb

```diff
@@ -59,25 +59,25 @@ def recurring
     { document_type: :world_location,             fields: [:content_id, :title, :schema_name, :locale, :analytics_identifier] },
   ].freeze
 
-  def root_reverse_links
+  def reverse_links
     REVERSE_LINKS.values
   end
 
   def reverse_link_type(link_type)
     REVERSE_LINKS[link_type.to_sym]
   end
 
-  def un_reverse_link_type(link_type)
+  def unreverse_link_type(link_type)
```

Comment:
yeah it's a crappy term since it's almost a double negative. 

It only unreverses reverse link types though, say if you passed it parent, it wouldn't swap that round to child for instance.